### PR TITLE
Correction of loading the Cart Rules creation page

### PR DESCRIPTION
**Pull request by claudioghirellifh**
## Issue Reference
No Reference

## Description
The page remained blank when attempting to access the cart rule creation page.

## How To Test This?
Navigate to the ADMIN:  Marketing > Promotions > Select the "Cart Rules" tab and Click on the Create Cart Rule Button

## Documentation
--

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php [hunk 1]
```diff
@@ -251,7 +254,8 @@ class="ltr:pr-10 rtl:pl-10"
                                     :key="index"
                                     :index="index"
                                     @onRemoveCondition="removeCondition($event)">
-                                />
+                                >
+                                </v-cart-rule-condition-item>
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php:258]: Hello @claudioghirellifh, 
We don't have to make all the changes. Just add the closing tag here like this (/>) and remove the (>) from line no. 256.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php (hunk 1 above)
Comment (claudioghirellifh) [reply packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php]: Hi @amit-webkul 
There really were some unnecessary changes. I hadn't realized they had gone along. I already made the correction. Thank you very much

File: packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php [hunk 2]
```diff
@@ -2,1100 +2,1100 @@
...
-            <div>
-                <x-admin::form
-                    :action="route('admin.marketing.promotions.cart_rules.store')"
-                    enctype="multipart/form-data"
-                >
+        @pushOnce('scripts')
+            <!-- v cart rules create form template -->
+            <script
+                type="text/x-template"
+                id="v-cart-rule-create-form-template"
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php:18]: @claudioghirellifh 
Here, some extra tab spaces are coming. Please remove the extra spaces.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php (hunk 2 above)
Comment (claudioghirellifh) [reply packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php]: @amit-webkul 
Using PhpStorm, it did not flag these extra spaces for me. Now, I performed the formatting using VSCode for .blade files.

**Pull request closed without merging**